netty/src/test/java/io/grpc/netty/NettyTestUtil.java

```diff
@@ -67,7 +67,9 @@ public Executor getObject() {
       inUse = true;
       return new Executor() {
         @Override
-        public void execute(Runnable var1) { }
+        public void execute(Runnable var1) {
+          var1.run();
```

Comment:
I wonder what lead to this change.

Reply:
This is an interesting issue. 2 bugs canceling each other out and giving a false positive. There was an internal discussion which I'll forward internally

Reply:
resolving this thread...